app/src/main/res/layout/listitem_advertisement_collection_start.xml

```diff
@@ -17,7 +17,7 @@
         <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:background="@drawable/gradient_cardview_alternative"
+            android:background="@drawable/gradient_cardview_accent"
```

Comment:
I see that you prefer blue because it looks more Bluetooth like.

But now the contrast “black/grey on blue” is very low. The text is super hard to read.